Makefile

```diff
@@ -257,7 +258,7 @@ bundle: $(OPM) $(YQ) manifests kustomize operator-sdk ## Generate bundle manifes
 	# Set desired operator image
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	# Update CSV
-	V="limitador-operator.v$(BUNDLE_VERSION)" $(YQ) eval '.metadata.name = strenv(V)' -i config/manifests/bases/limitador-operator.clusterserviceversion.yaml
```

Comment:
This should be BUNDLE_VERSION.

The `.metadata.name` cannot be `latest`, it needs to follow the format vX.Y.Z, and IMAGE_TAG can be `latest` o whatever.

Reply:
Ok, so it's either `v0.0.0` or `vX.Y.Z`